eth/config.go

```diff
@@ -33,6 +33,18 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+// DefaultFullGPOConfig contains default gasprice oracle settings for full node.
+var DefaultFullGPOConfig = gasprice.Config{
+	Blocks:     20,
+	Percentile: 60,
+}
+
+// DefaultFullGPOConfig contains default gasprice oracle settings for light client.
+var DefaultLightGPOConfig = gasprice.Config{
+	Blocks:     2,
+	Percentile: 50,
```

Comment:
Is there a reason to have a different `Percentile` value for light mode? You can also change it to 50 for full nodes if you think it's better.

Reply:
Fixed